.github/workflows/ci.yaml

```
@@ -416,7 +416,7 @@ jobs:
     timeout-minutes: 5
     strategy:
       matrix:
-        python-version: ["pypy3"]
+        python-version: ["pypy-3.7"]
```

```suggestion
        python-version: ["pypy-3.6"]
```
For the moment, I would suggest to stick with `pypy-3.6` since that is the equivalent to `pypy3`.
We can / should do the update to `pypy-3.7` after we removed support for Python 3.6.

---

I tested locally and `pypy-3.7` works. Since `pypy-3.6` is no longer being worked on I guess it makes sense to update as well? It shouldn't break CI.

---

I understand, but since we're only testing one pypy version, we should probably test that oldest supported one.
Besides that, such an update should be done in a separate PR. Dependency updates should be just that, nothing more.

---

Agreed that that should probably be done in a separate PR so I'll change this to use `3.6`.

However, note that `3.6` is no longer "supported". See [here](https://doc.pypy.org/en/latest/faq.html#which-python-version-2-x-does-pypy-implement). Or at least, the "main" version is now `3.7` which I think we should test against.

---

I'm not against adding `pypy-3.7`, just not here. And for the time being it should be in addition to `pypy-3.6` 😉

---

Fair enough 😄